appveyor.yml

```diff
@@ -8,20 +8,20 @@ install:
 - wget --no-check-certificate https://01.org/sites/default/files/libxml2-x86_64-3eaedba1b64180668fdab7ad2eba549586017bf3.zip
 - wget --no-check-certificate https://01.org/sites/default/files/libxml2-x86_64-debug-3eaedba1b64180668fdab7ad2eba549586017bf3.zip
 - wget --no-check-certificate https://01.org/sites/default/files/asio-1.10.6.tar.gz
+- wget --no-check-certificate https://raw.github.com/philsquared/Catch/master/single_include/catch.hpp --directory-prefix catch
 - 7z x libxml2-x86_64-3eaedba1b64180668fdab7ad2eba549586017bf3.zip
 - 7z x libxml2-x86_64-debug-3eaedba1b64180668fdab7ad2eba549586017bf3.zip
 - 7z x asio-1.10.6.tar.gz
 - 7z x asio-1.10.6.tar
+- set PREFIX_PATH=%APPVEYOR_BUILD_FOLDER%\libxml2-x86_64;%APPVEYOR_BUILD_FOLDER%\asio-1.10.6;%APPVEYOR_BUILD_FOLDER%\catch
```

Comment:
this uses the "release" version of the libxml2 for both debug and release builds.